src/Microsoft.Security.Utilities.Core/Marvin.NetFramework.cs

```diff
@@ -112,32 +112,32 @@ public static long ComputeHash(byte[] data, ulong seed, int offset, int length)
                 switch (length)
                 {
                     case 0:
-                        p0 += 0x80u;
-                        break;
+                    p0 += 0x80u;
```

Comment:
It's now not indenting cases with no braces. I suspect this is not what you want either.

Reply:
We'll fix this separately.